custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py

```diff
@@ -1520,28 +1542,30 @@ def clean_height(value):
 
 
 def clean_treatment_regimen(value):
-    if value is None:
+    if not value:
         return None
-    if value not in {
-        "inh_poly_mono",
-        "mdr_rr",
-        "short_regimen",
-        "mdr_rr_fq_sli",
-        "xdr",
-        "mixed_pattern",
-        "new_drug_mdr_rr_fq_sli",
-        "new_drug_xdr",
-        "new_fail_mdr",
-        "new_fail_xdr",
-        "new_mixed_pattern",
-    }:
-        raise FieldValidationFailure(value, "treatment reigmen")
-    return value
-    # return {
-    #     "Regimen for MDR/RR TB": "mdr_rr",
-    #     "Regimen for XDR TB": "xdr",
-    #     "Modified regimen for MDR/RR TB+ FQ/SLI resistance": "mdr_rr_fq_sli",
-    # }[value]
+    if value not in [
+        "Regimen for XDR TB",
+        "Regimen for MDR/RR TB",
+        "Modified Regimen for MDR/RR-TB + FQ/SLI resistance",
+        "Regimen with New Drug for MDR-TB Regimen + FQ/SLI resistance",
+        "Regimen with New Drug for XDR-TB",
+        "Modified regimen for mixed pattern resistance",
+        "Regimen for INH mono/poly resistant TB",
+        "Regimen with New Drug for failures of regimen for MDR TB",
+    ]:
+        raise FieldValidationFailure(value, "Treatment Regimen")
+
+    return {
+        "Regimen for XDR TB": "xdr",
+        "Regimen for MDR/RR TB": "mdr_rr",
+        "Modified Regimen for MDR/RR-TB + FQ/SLI resistance": "mdr_rr_fq_sli",
+        "Regimen with New Drug for MDR-TB Regimen + FQ/SLI resistance": "new_drug_mdr_rr_fq_sli",
+        "Regimen with New Drug for XDR-TB": "new_drug_xdr",
+        "Modified regimen for mixed pattern resistance": "mixed_pattern",
+        "Regimen for INH mono/poly resistant TB": "inh_poly_mono",
+        "Regimen with New Drug for failures of regimen for MDR TB": "new_fail_mdr",
+    }[value]
```

Comment:
Nit: you could avoid some duplication by doing:
```python
try:
    return {
        ...
    }[value]
except KeyError:
    raise FieldValidationFailure(...)
```
and dropping the first "if" bit.